dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestWrap.java

```diff
@@ -51,7 +51,11 @@ public void testIsWrapFor() {
             conn = Utils.getUserConnection();
             boolean result = conn.isWrapperFor(Connection.class);
             assertTrue("It is wrapper for this interface", result);
+            conn.close();
         } catch (SQLException e) {
+            if (conn !=null) {
+                conn.close();
```

Comment:
The Jenkins builds are complaining that this code has an uncaught exception of some sort. Could you take a look? Thanks.